addons/web_editor/static/src/components/html_history_dialog/html_history_dialog.js

```diff
@@ -0,0 +1,115 @@
+/** @odoo-module **/
+
+import { Dialog } from '@web/core/dialog/dialog';
+import { formatDateTime } from '@web/core/l10n/dates';
+import { useService } from "@web/core/utils/hooks";
+import { memoize } from "@web/core/utils/functions";
+import {Component, onMounted, useState, markup} from '@odoo/owl';
+import { _t } from "@web/core/l10n/translation";
+const { DateTime } = luxon;
+
+class HtmlHistoryDialog extends Component {
+    static template = "web_editor.HtmlHistoryDialog";
+    static components = { Dialog };
+    static props = {
+        close: Function,
+        restoreRequested: Function,
+        recordIds: Array,
+        versionnedFieldName: String,
+    };
+
+    state = useState({
+        revisionsData: [],
+        revisionContent: null,
+        revisionComparison: null,
+        revisionId: null,
+        asyncPending: true,
+    });
+    setup() {
+        this.size = 'xl';
+        this.title = _t("History");
+        this.orm = useService("orm");
+
+        onMounted(() => this.init());
+    }
+
+    async init() {
+        await this.getRecordData();
+        this.state.asyncPending = false;
+        await this.updateCurrentRevision(
+            this.state.revisionsData[0].id);
```

Comment:
This is not Python so here I'm sure this is not the correct way of linting that line.